stable/spartakus/templates/deployment.yaml

```diff
@@ -16,8 +16,8 @@ spec:
         release: "{{ .Release.Name }}"
     spec:
       containers:
-        - name: {{ template "fullname" . }}
-          image: "{{ .Values.image }}"
+        - name: {{ default "spartakus" .Values.containerName | quote }}
```

Comment:
You could use `.Chart.Name`? or `{{ template "name" . }}` which already uses `.Values.nameOverride` for overriding.